package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@nordcloud/gnui",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

This is a new addition (feature), minor version should be increased
```suggestion
  "version": "1.2.0",
```